lib/pdc/config.rb

```diff
@@ -118,7 +118,14 @@ def reset_base_connection
       faraday_config = {
         url:      api_url,
         headers:  PDC::Request.default_headers,
-        ssl:      ssl_config
+        ssl:      ssl_config,
+        # adding this request parameter is about changing how
```

Comment:
I like the comment, but miss the motivation of the change. Why is this changed? Was the culprit that the previous code only allowed sending single strings instead of lists?

Reply:
Yes, the previous codes just allowed string, add this can support both list and string type of the a field.